src/watchmaker/workers/salt.py

```diff
@@ -588,6 +593,7 @@ def __init__(self, *args, **kwargs):
             'policycoreutils-python',
             'selinux-policy-targeted',
             'salt-minion',
+            'python-pip'
```

Comment:
```suggestion
            'python-pip',
```

Reply:
One complicating factor is that `python-pip` is for python2. This will work when installing the python2 versions of salt. It will not work when installing the python3 versions of salt. And, it will install an entire python2 system-level runtime if one is not installed (such as on EL8), which would be way undesirable if using python3 salt.

I don't have another suggestion just yet, but this needs some noodling...